ecs_api.tf

```diff
@@ -93,9 +93,10 @@ data "template_file" "api_service_container_definitions" {
   template = file("templates/api_service_task_definition.tpl")
 
   vars = {
-    api_image_uri        = "${aws_ecr_repository.api.repository_url}:latest"
+    api_image_uri        = "${aws_ecr_repository.api.repository_url}:${var.api_container_tag}"
```

Comment:
@segfault Guessing this is a redundant line ?

Can we change the **container** in the var names to **image** ?

Is the idea that you could set the migrations_image_repo_url ? If so will the created ECR repos be ignored if you use a different image registry ?

Reply:
Works for me